snapcraft/parts/grammar.py

```diff
@@ -64,12 +64,12 @@ def process_parts(
     :param yaml_data: unprocessed snapcraft.yaml.
     :returns: process snapcraft.yaml.
     """
+
+    def self_check(x: Any) -> bool:
+        return x == x  # pylint: disable=comparison-with-itself  # noqa PLR0124
```

Comment:
pylint: C0103: Argument name "x" doesn't conform to snake_case naming style (invalid-name)

Reply:
Renamed to `value`. I wonder why I didn't get this when I linted locally 🤔